[mielecloud] Integration tests failure

It is failing apparently sysrematically.
Build on Windows:
```
[INFO] --- bnd-testing-maven-plugin:6.2.0:testing (default) @ org.openhab.binding.mielecloud.tests ---
[INFO] Matching glob *
[INFO] Testing D:\dev\openhab3\git\openhab-addons\itests\org.openhab.binding.mielecloud.tests\itest.bndrun:
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Thing creation failed because there was no binding available that supports the thing.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Cannot create bridge: E-mail address is missing.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Cannot create bridge: E-mail address is missing.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Bridge exists but has no handler.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Cannot create bridge: Bridge UID is missing.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.sse.SseStreamParser] WARN : SSE connection failed unexpectedly: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.sse.SseStreamParser] WARN : SSE connection failed unexpectedly: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
...
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.sse.SseStreamParser] WARN : SSE connection failed unexpectedly: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Cannot modify non existing bridge: Could neither add bridge via inbox nor find existing bridge.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Cannot create bridge: Bridge UID 'null' is malformed.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.CreateBridgeServlet] WARN : Cannot create bridge: Bridge UID is missing.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Failed to create authorization URL.
org.openhab.binding.mielecloud.internal.auth.OAuthException:
        at org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet.getRedirectionDestination(ForwardToLoginServlet.java:113)
        at org.openhab.binding.mielecloud.internal.config.servlet.AbstractRedirectionServlet.doGet(AbstractRedirectionServlet.java:52)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:618)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
        at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:602)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1434)
        at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:294)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:82)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:386)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
        at java.base/java.lang.Thread.run(Thread.java:834)
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing e-mail address.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing client ID.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing bridge ID.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing client ID.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Cannot begin new authorization process while another one is still running.
TEST org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServletTest#whenAuthorizationCannotBeBegunThenTheBrowserIsRedirectedToThePairSiteAndAWarningIsDisplayed() <<< ERROR: expected: <true> but was: <false>
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
        at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
        at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:35)
        at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:179)
        at org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServletTest.whenAuthorizationCannotBeBegunThenTheBrowserIsRedirectedToThePairSiteAndAWarningIsDisplayed(ForwardToLoginServletTest.java:59)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
        at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at aQute.tester.bundle.engine.BundleDescriptor.executeChild(BundleDescriptor.java:49)
        at aQute.tester.bundle.engine.BundleEngine.lambda$executeBundle$7(BundleEngine.java:120)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
        at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
        at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
        at aQute.tester.bundle.engine.BundleEngine.executeBundle(BundleEngine.java:120)
        at aQute.tester.bundle.engine.BundleEngine.lambda$executeBundle$8(BundleEngine.java:133)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
        at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
        at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
        at aQute.tester.bundle.engine.BundleEngine.executeBundle(BundleEngine.java:133)
        at aQute.tester.bundle.engine.BundleEngine.lambda$execute$5(BundleEngine.java:100)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
        at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
        at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
        at aQute.tester.bundle.engine.BundleEngine.execute(BundleEngine.java:100)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
        at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
        at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
        at aQute.tester.junit.platform.Activator.test(Activator.java:439)
        at aQute.tester.junit.platform.Activator.automatic(Activator.java:344)
        at aQute.tester.junit.platform.Activator.run(Activator.java:216)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at aQute.launcher.Launcher.launch(Launcher.java:450)
        at aQute.launcher.Launcher.run(Launcher.java:184)
        at aQute.launcher.Launcher.main(Launcher.java:160)
        at aQute.launcher.pre.EmbeddedLauncher.executeWithRunPath(EmbeddedLauncher.java:170)
        at aQute.launcher.pre.EmbeddedLauncher.findAndExecute(EmbeddedLauncher.java:135)
        at aQute.launcher.pre.EmbeddedLauncher.main(EmbeddedLauncher.java:52)
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing e-mail address.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing client secret.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Failed to create authorization URL: There was no ongoing authorization although we just started one.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Passed bridge ID 'genesis!' is invalid.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing client secret.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ForwardToLoginServlet] WARN : Request is missing bridge ID.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: unknown_oauth_2_error
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: server_error
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: invalid_request
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: unauthorized_client
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: access_denied
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Code is null
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : State is null
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Failed to complete authorization: There is no ongoing authorization or it timed out
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: invalid_scope
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Failed to complete authorization.
org.openhab.binding.mielecloud.internal.auth.OAuthException:
        at org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet.getRedirectionDestination(ResultServlet.java:82)
        at org.openhab.binding.mielecloud.internal.config.servlet.AbstractRedirectionServlet.doGet(AbstractRedirectionServlet.java:52)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:618)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
        at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:799)
        at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
        at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
        at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:602)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235)
        at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1624)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233)
        at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1434)
        at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:294)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188)
        at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1594)
        at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186)
        at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:82)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127)
        at org.eclipse.jetty.server.Server.handle(Server.java:516)
        at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:388)
        at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:633)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:380)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:277)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105)
        at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:338)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:315)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:173)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:386)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
        at java.base/java.lang.Thread.run(Thread.java:834)
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: unsupported_response_type
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.ResultServlet] WARN : Received error response: temporarily_unavailable
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.SuccessServlet] WARN : Success page is missing e-mail address.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.SuccessServlet] WARN : Success page is missing bridge UID.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.SuccessServlet] WARN : Success page is missing bridge UID.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.SuccessServlet] WARN : Success page is missing e-mail address.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.config.servlet.SuccessServlet] WARN : Success page received malformed bridge UID '!genesis'.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.discovery.ThingDiscoveryService] WARN : Found no matching thing type for device type VACUUM_DRAWER
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.CoffeeSystemThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.CoffeeSystemThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:genesis:000124430017 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.CoolingDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.CoolingDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.DishWarmerDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.DishWarmerDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.DishwasherDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.DishwasherDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.DryerDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.DryerDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.HobDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.HobDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.HoodDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.HoodDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot remove listener: The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot remove listener: The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot disconnect from SSE stream: The Miele cloud service is not available.
org.openhab.core.thing [org.openhab.core.thing.internal.ThingManagerImpl] WARN : Cannot unregister handler. No handler factory for thing 'mielecloud:account:genesis' found.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot remove listener: The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot remove listener: The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot disconnect from SSE stream: The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:000124430016 tried updating channel remote_control_can_be_started although the handler was already disposed.
org.openhab.core.thing [org.openhab.core.thing.binding.BaseThingHandler] WARN : Handler CoffeeSystemThingHandler of thing mielecloud:coffee_system:000124430016 tried updating channel remote_control_can_be_stopped although the handler was already disposed.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot re-emit device state for device '000124430016': The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.auth.OpenHabOAuthTokenRefresher] WARN : Token refreshing was requested but there is no token refresh listener registered!
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.UnavailableMieleWebservice] WARN : Cannot connect to SSE stream: The Miele cloud service is not available.
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.webservice.sse.SseStreamParser] WARN : SSE connection failed unexpectedly: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.OvenDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.OvenDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.RoboticVacuumCleanerDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.RoboticVacuumCleanerDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.WashingDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.WashingDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.WineStorageDeviceThingHandler] WARN : Failed to perform 'STOP' operation for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
org.openhab.binding.mielecloud [org.openhab.binding.mielecloud.internal.handler.WineStorageDeviceThingHandler] WARN : Failed to set light state to 'true' for device '000124430017'.
org.openhab.binding.mielecloud.internal.webservice.exception.MieleWebserviceException
[INFO] 1 Error(s)
[ERROR] Error   : Exit code remote process 1: java -cp D:/dev/openhab3/git/openhab-addons/itests/org.openhab.binding.mielecloud.tests/target/test/tmp/testing/itest/cnf/cache/6.2.0/bnd-cache/biz.aQute.launcher/biz.aQute.launcher.pre.jar -Dlauncher.properties=D:\dev\openhab3\git\openhab-addons\itests\org.openhab.binding.mielecloud.tests\target\test\tmp\testing\itest\generated\launch12662080119462504174.properties -Dorg.osgi.service.http.port=55029 --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.security=ALL-UNNAMED --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.base/java.time=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.desktop/java.awt.font=ALL-UNNAMED --add-opens=java.naming/javax.naming.spi=ALL-UNNAMED --add-opens=java.rmi/sun.rmi.transport.tcp=ALL-UNNAMED -ea -Dlauncher.runpath=D:/dev/openhab3/git/openhab-addons/itests/org.openhab.binding.mielecloud.tests/target/test/tmp/testing/itest/cnf/cache/6.2.0/org.openhab.core.bom.runtime-index/org.eclipse.osgi-3.17.200.v20220215-2237.jar,D:/dev/openhab3/git/openhab-addons/itests/org.openhab.binding.mielecloud.tests/target/test/tmp/testing/itest/cnf/cache/6.2.0/bnd-cache/biz.aQute.launcher/biz.aQute.launcher-6.2.0.jar aQute.launcher.pre.EmbeddedLauncher
```

--- a/itests/org.openhab.binding.mielecloud.tests/src/main/java/org/openhab/binding/mielecloud/internal/config/servlet/ForwardToLoginServletTest.java
+++ b/itests/org.openhab.binding.mielecloud.tests/src/main/java/org/openhab/binding/mielecloud/internal/config/servlet/ForwardToLoginServletTest.java
@@ -57,7 +57,7 @@ public class ForwardToLoginServletTest extends AbstractConfigFlowTest {
         assertTrue(maybePairAccountSite.contains(
                 "Go to <a href=\"https://www.miele.com/f/com/en/register_api.aspx\">the Miele developer portal</a> to obtain your"));
         assertTrue(maybePairAccountSite.contains(
-                "There is an authorization ongoing at the moment. Please complete that authorization prior to starting a new one or try again in 3 minutes."));
+                "There is an authorization ongoing at the moment. Please complete that authorization prior to starting a new one or try again"));
     }
 
     @Test

